apps/mobile/src/components/InsideRouter/components/MarketplaceScreen/components/EmptyListPlaceholder.tsx

```diff
@@ -106,7 +106,8 @@ function EmptyListPlaceholder({refreshing, onRefresh}: Props): JSX.Element {
         if (minutesTillOffersDisplayed > 0) {
           setMinutesTillOffersDisplayed(minutesTillOffersDisplayed - 1)
         }
-        void refreshOffers()
+        // console.log('Refreshing offers from emptyListPlaceholder')
+        // void refreshOffers()
```

Comment:
Should this happen? 
Also, the refresh should be disabled, once we have some offers to prevent unnecessary rendering